core/templates/domain/state_card/state-card.model.ts

```diff
@@ -143,13 +153,16 @@ export class StateCard {
     this._completed = false;
   }
 
-  getInteractionInstructions(): string {
+  // Some interaction specifications do not have instructions.
+  getInteractionInstructions(): string | null {
```

Comment:
Should we then return `null` instead of `''` if `interactionId` doesn't exists ?